app/middleware/authentication.js

```diff
@@ -146,9 +148,29 @@ function verifyProfessor(req, res, next) {
   });
 }
 
+/**
+ * Pre-req: none
+ * Ensure that the Team with a valid link_code exists and attaches it to req.user.
+ */
+function validateTeam(req, res, next) {
```

Comment:
We should probably follow the established convention of other functions. Since this is attaching the `team` to the request, consider renaming this function to `validateTeamAttachTeam`